worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -272,6 +273,36 @@ public void smoothBrush(Player player, LocalSession session,
         ToolCommands.sendUnbindInstruction(player, UNBIND_COMMAND_COMPONENT);
     }
 
+    @Command(
+        name = "snowsmooth",
+        desc = "Choose the snow terrain softener brush",
+        descFooter = "Example: '/brush snowsmooth 5 1 3'"
+    )
+    @CommandPermissions("worldedit.brush.snowsmooth")
+    public void snowSmoothBrush(Player player, LocalSession session,
+                                @Arg(desc = "The radius to sample for softening", def = "2")
+                                    double radius,
+                                @Arg(desc = "The number of iterations to perform", def = "4")
+                                    int iterations,
+                                @Arg(desc = "The number of snow blocks under snow", def = "1")
+                                    int snowBlockCount,
+                                @Arg(desc = "The mask of blocks to use for the heightmap", def = "")
+                                    Mask mask) throws WorldEditException {
+        worldEdit.checkMaxBrushRadius(radius);
+
+        BrushTool tool = session.getBrushTool(player.getItemInHand(HandSide.MAIN_HAND).getType());
+        tool.setSize(radius);
+        tool.setBrush(new SnowSmoothBrush(iterations, snowBlockCount, mask), "worldedit.brush.snowsmooth");
+
+        player.printInfo(TranslatableComponent.of(
+                "worldedit.brush.snowsmooth.equip",
+                TextComponent.of((int) radius),
+                TextComponent.of(iterations),
+                TextComponent.of(mask == null ? "any block" : "filter"),
```

Comment:
This should probably be translatable. They're nestable, so you just need to add keys for it and change the component.